web/src/main/webapp/package-lock.json

```diff
@@ -0,0 +1,511 @@
+{
```

Comment:
As far as I understand, it makes sure that everyone building the front-end uses the same package versions. It is advised that this file is checked in to VCS. From the [npm docs](https://docs.npmjs.com/files/package-lock.json):
> package-lock.json is automatically generated for any operations where npm modifies either the node_modules tree, or package.json. It describes the exact tree that was generated, such that subsequent installs are able to generate identical trees, regardless of intermediate dependency updates.